features/list_of_posts.feature

```diff
@@ -0,0 +1,19 @@
+Feature: List of Posts
+  As a reader
+  I want to read posts from my favorite blogger
+  So that I can learn something
+
+
+  Scenario: Clicking enter after URL is entered
```

Comment:
This isn't really capturing what the scenario is. Looking at your steps, I'd expect the scenario to be named something like `Home page displays all posts` or `All published posts are visible`.